config/locales/views.fr.yml

```
@@ -269,6 +269,7 @@ fr:
     new_solicitation_form:
       attributes:
         help:
+          siret_assistive_hint: Lorsque les résultats de l'auto-complétion sont disponibles, utilisez les flèches haut et bas pour les examiner et entrer pour sélectionner une option. Les utilisateurs d'appareils tactiles peuvent explorer par le toucher ou avec des gestes de glissement.
```

Si on en est à pinailler, 
```suggestion
          siret_assistive_hint: Lorsque les résultats de l’auto-complétion sont disponibles, utilisez les flèches haut et bas pour les examiner et Entrée pour sélectionner une option. Les utilisateurs d’appareils tactiles peuvent explorer par le toucher ou avec des gestes de glissement.
```

Il faudrait que je trouve ou que je fasse un script pour remplacer les `'` par des `’` dans les fichiers de localisation.

---

Je sais même pas comment faire la distinction  ' et ’  sur mon clavier. Ni c'est quoi la règle. (bouh la mauvaise typographe)

---

Me suis contentée de traduire la version anglaise, pour info. Je mettrais bien quand même qqch, je me méfie des "c’est à peu près évident"

---

Oui oui, j’avais vu la version anglaise et le commentaire de Lucien. Mais en fait:
* avec un clavier souris, les gens vont utiliser une souris, et ça va marcher; s’ils utilisent les touches, tant mieux pour eux mais ça n’a rien d’obligatoire.
* Sur tactile, “explore by touch or swipe gestures”, ben… y a-t-il d’autres façons de faire ?

---

là, on parle de gens qui n'utilisent que le clavier, sur ordi. C'est pour l'accessibilité (si ça se trouve c'était clair pour toi mais ton commentaire me met le doute).

---

Ah ben c’était pas clair du tout pour moi en fait. 👍